dev/travis/script.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Copyright © 2015 Magento. All rights reserved.
+# See COPYING.txt for license details.
+
+set -e
+export PATH="./../../../vendor/bin:$PATH"
```

Comment:
Moving path to `vendor` here looks risky. In case another case will be added for tests with different hierarchy, it may not work unexpectedly.